src/spaces/cam16.js

```diff
@@ -195,7 +195,7 @@ export function fromCam16(cam16, env) {
 		hRad = constrain(cam16.h) * deg2rad;
 	}
 	else {
-		h_rad = invHueQuadrature(H) * deg2rad;
+		hRad = invHueQuadrature(cam16.H) * deg2rad;
```

Comment:
@facelessuser I am not confident in these changes, but the new lint rules highlighted that `h_rad` and `H` were both undefined here. Can you clarify the intention?

Reply:
Yeah, `h_rad` was a typo on my part, it should be `hRad` as you've done. We aren't currently generating the reverse transform with hue quadrature, so this wasn't caught.

`H` is not likely to be undefined as we check to make sure `h` or `H` is defined on line 183. So one of the must be defined at this point, but it may confuse the checker.

Reply:
Thanks! So it sounds like this is good to go.

> `H` is not likely to be undefined as we check to make sure `h` or `H` is defined on line 183. So one of them must be defined at this point, but it may confuse the checker.

`H` was undefined (likely a typo) -- this changes it to `cam16.H`, which is defined.

Reply:
Oh, `H` was undefined! 🤦🏻 Sorry, this all looks correct.